VideoLocker/src/main/java/org/edx/mobile/view/FindCoursesActivity.java

```diff
@@ -23,9 +23,19 @@ protected void onCreate(Bundle savedInstanceState) {
         }catch(Exception e){
             logger.error(e);
         }
+
+        reload();
+    }
+
+    @Override
+    protected void onOnline() {
+        super.onOnline();
+        reload();
+    }
+
+    private void reload() {
```

Comment:
Can this function be renamed to <code>load()</code> instead of <code>reload()</code> as it is being called from <code>onCreate()</code> as well.